docs/usage/user.asciidoc

```diff
@@ -0,0 +1,396 @@
+`event.category` and `event.type` with regards to IAM activity. Make sure to read
+the <<ecs-category-field-values-reference, Categorization section>> to see all allowed
+values, and read more about `event.kind` and `event.outcome`.
+
+NOTE: IAM activity is a bit particular in that events are expected to be assigned 2 event types.
+One of them indicates the type of activity (creation, deletion, change, etc.),
+and the other indicates whether a user or a group is the target of the management activity.
+
+Many sections of the examples below are elided, in order to focus on the categorization
+of the events.
+
+Creation of group "test-group":
+
+```JSON
+{
+  "event": {
+    "kind": "event",
+    "category": ["iam"], <1>
+    "type": ["group", "creation"], <2>
+    "outcome": "success"
+  },
+  "group": { "name": "test-group", ... },
+  "user": { ... },
+  "related": { "user": [ ... ] }
+}
+```
+<1> Category "iam"
+<2> Both relevant event types to a group creation
+
+Adding "test-user" to "test-group":
+
+```JSON
+{
+  "event": {
+    "kind": "event",
+    "category": ["iam"], <1>
+    "type": ["user", "change"], <2>
+    "action": "user added to group", <3>
+    "outcome": "success"
+  },
+  "user": {
+    ...
+    "target": { <4>
+      "name": "test-user",
+      "group": { "name": "test-group" }
+    }
+  },
+  "related": { "user": [ ... ] }
+}
+```
+<1> Category "iam"
+<2> Both relevant event types to a user modification
+<3> `event.action` is not a categorization field, and has no mandated value. It can be populated based on source event details or by a pipeline, to ensure the event captures all subtleties of what's happening.
+<4> How to use all possible user fields is detailed below.
+
+[discrete]
+[[ecs-user-usage-field-reuse]]
+===== Field reuse
+
+The user fields can be reused (or appear) in many places across ECS. This makes
+it possible to capture many users relevant to a single event.
+
+Here's the full list of places where the user fields can appear:
+
+* `user.*`
+* `user.effective.*`
+* `user.target.*`
+* `user.changes.*`
+* `source.user.*`
+* `destination.user.*`
+* `client.user.*`
+* `server.user.*`
+* `host.user.*` (<<ecs-user-usage-deprecations,deprecated>>)
+
+Let's go over the meaning of each.
+
+The examples below will only populate `user.name` and sometimes `user.id` inside
+the various `user` nestings, for readability.
+However in implementations, unless otherwise noted, all `user` fields that can
+reasonably be populated in each location should be populated.
+
+[discrete]
+[[ecs-user-usage-user-at-root]]
+====== User fields at the Root of an Event
+
+The user fields at the root of an event are used to capture the user
+performing the main action described by the event. This is especially important
+when there's more than one user present on the event. `user.*` fields at the root
+of the event represent the user performing the action.
+
+In many cases, events that only mention one user should populate the user fields
```

Comment:
A high-level summary of the guidance covered could be a useful companion quick-reference to the more detailed explanations. I could see it both early in the doc as well as towards the end as a summary.

Examples of quick guidance:

* Always populate `user.*` in addition to more purpose-specific fields when one or more users are present in an event. If more than one user, the user performing the main action populates `user.*`. 
* Any users present in the event should also be captured in `related.user`
* etc. etc.

Reply:
Good idea!

Reply:
I feel like this is not a blocker to this PR. WDYT?

Perhaps we could note down some of the points for this quick guidance in a separate issue, and address separately?

Reply:
Agreed, it's not a blocker here. We can capture and address later on 😄 .